src/test/java/core/basesyntax/SimpleCalculatorTest.java

```diff
@@ -0,0 +1,89 @@
+package core.basesyntax;
+
+import org.junit.Assert;
+import org.junit.Test;
+import static java.lang.Float.NaN;
```

Comment:
```suggestion
import static java.lang.Float.NaN;

import org.junit.Assert;
import org.junit.Test;
```
check out the article I shared earlier
static imports should be places first and separated with a blank line